ethstorage/p2p/protocol/sync_test.go

```diff
@@ -1,6 +1,8 @@
 // Copyright 2022-2023, EthStorage.
 // For license information, see https://github.com/ethstorage/es-node/blob/main/LICENSE
 
+//go:build !ci
```

Comment:
Since you've add the !ci flag, I suggest that remove the related following changes in this file and leave it with Ping to  fix it